ChangeFeedLabSolution/Startup.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+using Microsoft.Extensions.Azure;
+using System;
+
+[assembly: FunctionsStartup(typeof(ChangeFeedFunction.Startup))]
+namespace ChangeFeedFunction
+{
+    public class Startup : FunctionsStartup
+    {
+        public override void Configure(IFunctionsHostBuilder builder)
+        {
+            builder.Services.AddAzureClients(builder =>
+            {
+                builder.AddEventHubProducerClient(Environment.GetEnvironmentVariable("EventHubNamespaceConnection"), Environment.GetEnvironmentVariable("EventHubName"));
```

Comment:
What happens if we want to read these values from `local.settings.json` where they're also specified?   We should be using the Functions configuration types.   Please see:  https://docs.microsoft.com/azure/azure-functions/functions-dotnet-dependency-injection#working-with-options-and-settings

Reply:
@jsquire -  We can also read these values from `local.settings.json` using `Environment.GetEnvironmentVariable("")`